README.md

```diff
@@ -5,27 +5,51 @@
 [![Test Coverage](https://codeclimate.com/github/18F/federalist/badges/coverage.svg)](https://codeclimate.com/github/18F/federalist/coverage)
 [![Dependency Status](https://gemnasium.com/badges/github.com/18F/federalist.svg)](https://gemnasium.com/github.com/18F/federalist)
 
-***Under active development. Everything is subject to change. Learn more at the [documentation site](https://federalist-docs.18f.gov/). Interested in talking to us? [Join our public chat room](https://chat.18f.gov/).***
+***Federalist is updated regularly.  Contact us via [email](federalist-support@gsa.gov) or [Join our public chat room](https://chat.18f.gov/) to talk to us and stay informed. Or, check our our [documentation](https://federalist-docs.18f.gov/).***
 
-Federalist is a unified interface for publishing static government websites. It automates common tasks for integrating GitHub and Amazon Web Services to provide a simple way for developers to launch new static websites or more easily manage existing static websites. This repo is home to "federalist-core" - a Node.js app that allows government users to add and configure their Federalist sites.
+## About Federalist
+
+Federalist helps government agencies publish basic websites quickly and seamlessly.  It provides a simple interface for managing simple website content that is compliant with federal guidelines.  More complex tasks are automated behind the scenes, providing a way for users to more easily create and launch government websites.  Federalist is built on top of Github and Amazon Web Services.   
+
+## Examples
+
+Partner agencies across the federal government use Federalist to host websites.  A few examples include: 
+
+- [College Scorecard] (https://collegescorecard.ed.gov/)
+- [Natural Resources Revenue Data] (https://revenuedata.doi.gov/)
+- [NSF Small Business Innovation Research program] (https://seedfund.nsg.gov)
+
+More examples can be found [here] (https://federalist-docs.18f.gov/pages/about-federalist/example-sites/). 
 
 ## Getting started
 
+This section will walk you through how to install Federalist locally onto your own machine or within your own environment.  Depending on your familiarity with the dependencies and process, this may take anywhere between an hour or two to the better part of a day.  If you need help, [contact us](federalist-support@gsa.gov). 
```

Comment:
I'm not totally sure we want to offer external support for development setup.

Reply:
I would find this frustrating if I didn't know where to go other than the docs for help, and I was a prospective customer.  But I understand your point.  Do we have a sense of how often this might be likely to occur - is it worth testing out?

Reply:
Hrm, I don't think a prospective customer would be setting up a development instance of the Federalist application though. It's really only for 18F developers, and maybe security researchers or extremely curious open source aficionados.

Reply:
Agreed with @jseppi - Federalist is not made to be run by customers. There's no functionality used by customers that involves running locally.

Reply:
We should put content like this in the TTS handbook.

Reply:
@wslack Are you suggesting that this content move to the TTS handbook **and** this section of the Federalist README should point people there? Also, it sounds like y'all are contemplating potentially moving this entire section of instructions. Is that from **Getting started** all the way through **Testing and linting**?